src/main/java/com/google/firebase/FirebaseOptions.java

```diff
@@ -207,6 +210,16 @@ public int getReadTimeout() {
     return readTimeout;
   }
 
+  /**
+   * Returns the write timeout in milliseconds, which is applied to outgoing REST calls
```

Comment:
Since this is a case for "that" instead of which, let's just rephrase to something like:

"Returns the write timeout applied to outgoing REST calls in milliseconds."

If you think we can go ahead and assume "by the SDK."